doc/nrf/known_issues.rst

```diff
@@ -119,6 +119,11 @@ CIA-463: Wrong network mode parameter reported to cloud
   The network mode string present in ``deviceInfo`` (nRF Cloud) and ``dev`` (Azure IoT Hub and AWS IoT) JSON objects that is reported to cloud might contain wrong network modes.
   The network mode string contains the network modes that the modem is configured to use, not what the modem actually connects to the LTE network with.
 
+.. rst-class:: v1-8-0 v1-9-0 v1-9-1
```

Comment:
```suggestion
.. rst-class:: v1-9-1 v1-9-0 v1-8-0
```

Reply:
This issue only applies for 1.9.0 and 1.9.1 if I'm not mistaken. @tokangas please confirm

Reply:
> This issue only applies for 1.9.0 and 1.9.1 if I'm not mistaken. @tokangas please confirm

Yes, only v1.9.0 and v1.9.1 are affected.

Reply:
Removed v1.8.0.